scripts/ironic-common.sh

```diff
@@ -7,6 +7,27 @@ PROVISIONING_INTERFACE="${PROVISIONING_INTERFACE:-}"
 PROVISIONING_IP="${PROVISIONING_IP:-}"
 PROVISIONING_MACS="${PROVISIONING_MACS:-}"
 IPXE_CUSTOM_FIRMWARE_DIR="${IPXE_CUSTOM_FIRMWARE_DIR:-/shared/custom_ipxe_firmware}"
+CUSTOM_CONFIG_DIR="${CUSTOM_CONFIG_DIR:-/conf}"
+CUSTOM_DATA_DIR="${CUSTOM_DATA_DIR:-/data}"
+export IRONIC_CONF_DIR="${CUSTOM_CONFIG_DIR}/ironic"
```

Comment:
Group these per component. Alphabetical sort should do the trick.

Reply:
I have done it everywhere where the dependency chain allowed it and also separated the file paths from the directory paths.